cli/resolver.rs

```diff
@@ -1065,7 +1053,7 @@ impl SloppyImportsResolver {
     }
 
     let probe_paths: Vec<(PathBuf, SloppyImportsResolutionReason)> =
-      match (stat_sync)(&path) {
+      match self.stat_cache.stat_sync(&path) {
```

Comment:
One thing we're probably going to have to do with this change is not use a stat cache because it will get out of date.

Another point to note (that I mentioned on the last PR, but worth re-iterating to keep in mind) is that sloppy import resolution may change when a file is added or deleted. Probably in the future (I wouldn't suggest as part of this PR), we'll need to invalidate resolved resolution more when using sloppy imports.